### DISCUSSION TITLE
GitHub Actions Failing to Build .NET 9.0 Project

### DISCUSSION THREAD

**Discussion by ZjzMisaka** [community/community#160529] [Actions] [answered]
Hi, 

I recently updated my project to target .NET 9.0 after its release. I modified the csproj file to set the TargetFramework to .NET 9.0 and successfully built the project on my local machine. However, after committing the changes, the GitHub Actions workflow fails during the build step with the following error:
```
The current .NET SDK does not support targeting .NET 9.0. Either target .NET 8.0 or lower, or use a version of the .NET SDK that supports .NET 9.0. Download the .NET SDK from https://aka.ms/dotnet/download
```
This suggests that the version of the .NET SDK being used in the GitHub Actions environment does not yet support .NET 9.0.

Questions: 
1. Is .NET 9.0 supported in GitHub Actions?
2. Do I need to configure my workflow specifically for .NET 9.0?

My workflow: 
https://github.com/ZjzMisaka/PowerThreadPool/blob/main/.github/workflows/test.yml

**Comment by patrick-dmxc** [comment]
In my Action its also not running 
https://github.com/DMXControl/RDMSharp

**Reply by ZjzMisaka** [reply, marked as answer]
hi, the issue solved in https://github.com/actions/setup-dotnet/issues/562

**Reply by patrick-dmxc** [reply]
Show me how it works on a Matrix

      
jobs:
  build:
    strategy:
      matrix:
        os: [ubuntu-latest, windows-latest, macos-latest]
        dotnet-version: [ '6.0', '7.0', '8.0', '9.0' ]
    runs-on: ${{ matrix.os }}
